math/prime/millerrabinprimalitytest_test.go

```diff
@@ -23,7 +18,17 @@ func TestMillerRabinTest(t *testing.T) {
 	}
 	for _, test := range tests {
 		t.Run(test.name, func(t *testing.T) {
-			output, err := MillerRabinTest(test.input, test.rounds)
+			output, err := MillerRabinProbabilistic(test.input, test.rounds)
+			if err != test.err {
+				t.Errorf("For input: %d, unexpected error: %v, expected error: %v", test.input, err, test.err)
+			}
+			if output != test.expected {
+				t.Errorf("For input: %d, expected: %v", test.input, output)
+			}
+		})
+
+		t.Run(test.name, func(t *testing.T) {
+			output, err := MillerRabinDeterministic(test.input)
```

Comment:
separate out the tests - they should be independent

Reply:
Done.